charts/k8s-monitoring/docs/ScrapeApplicationMetrics.md

```diff
@@ -99,24 +99,24 @@ This chart automatically creates three components that you can utilize:
 * `discovery.kubernetes.pods` - Discovers all pods in the cluster
 
 These are
-all [`discovery.kubernetes`](https://grafana.com/docs/agent/latest/flow/reference/components/discovery.kubernetes/)
+all [`discovery.kubernetes`](https://grafana.com/docs/alloy/latest/reference/components/discovery.kubernetes/)
 components which gather all the specific resources using the Kubernetes API. From here, you want to refine the search
 to the service or the Pod that you want.
 
 ### Service discovery
 
 Since you don't want to scrape every service in your Cluster, use rules to select your specific service based
 on its name, namespace, labels, port names or numbers, and many other variables.
-To do so, use 
-a [`discovery.relabel`](https://grafana.com/docs/agent/latest/flow/reference/components/discovery.relabel/)
+To do so, use
+a [`discovery.relabel`](https://grafana.com/docs/alloy/latest/reference/components/discovery.relabel/)
 component and add one or more rules, using special meta-labels that are set automatically by the
 `discovery.kubernetes` component.
 
 Here is an example component that we've named "blue_database_service". This component takes the list of all services
 from `discovery.kubernetes.services` and filters to a service named "database", in the namespace "blue", with the port
 named "metrics":
 
-```river
+```alloy
```

Comment:
If this documentation is going to be rendered on grafana.com/docs, then this is the correct Markdown tag to use. However, if this is for GitHub only, then `alloy` refers to the [specification language](https://en.wikipedia.org/wiki/Alloy_(specification_language)) from 1997.

Reply:
Ahh geez... Ok. This is just in GitHub. Is there one that we should use instead? Or just plain text?

Reply:
I've been using `grafana-alloy` for GitHub-only docs, but since GitHub doesn't recognize that, blank/plaintext is fine too 🤷

Reply:
Done.